src/analytics/src/main/scala/osmesa/analytics/stats/package.scala

```diff
@@ -46,6 +65,12 @@ package object stats {
     isCoastline(tags) or
     isPOI(tags) as 'isInterestingWay
 
+  def isLanduse(tags: Column): Column =
+    tags.getItem("landuse").isNotNull as 'isLanduse
+
+  def isNatural(tags: Column): Column =
+    tags.getItem("natural").isNotNull as 'isNatural
+
```

Comment:
Is there a problem with double counting coastlines in the natural category?  How about railways in landuse?  I doubt the separate category should be removed, but should the more catchall categories have those tags omitted?

Reply:
I asked the client about this. They're ok with double counting these cases.

Reply:
Good catch though.